File: VeresiyeTakip/VeresiyeTakip/bin/Debug/ru/g0hz0nxs.4w3/ItemTemplates/VisualBasic/Extensibility/Toolbox/1033/VBWinFormsToolboxItemTemplate/ToolboxControl.vb
Imports System.Windows.Forms

<ProvideToolboxControl("$rootnamespace$.$toolBoxControl$", False)>
Public Class $toolBoxControl$

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        MessageBox.Show(String.Format(System.Globalization.CultureInfo.CurrentUICulture, "$message$", Me.ToString()))
    End Sub

End Class

File: VeresiyeTakip/VeresiyeTakip/bin/Debug/ru/g0hz0nxs.4w3/ItemTemplates/CSharp/Extensibility/Toolbox/1033/WinFormsToolboxItem/ToolboxControl.cs
using System.Globalization;
using System;
using System.Windows.Forms;

namespace $rootnamespace$
{
    [ProvideToolboxControl("$rootnamespace$.$toolBoxControl$", false)]
    public partial class  $toolBoxControl$ : UserControl
    {
        public  $toolBoxControl$()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            MessageBox.Show(string.Format(CultureInfo.CurrentUICulture, "$message$", this.ToString()));
        }
    }
}
